src/test/java/com/gildedrose/ItemsControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.gildedrose;
+
+import com.gildedrose.goblins_grotto.Item;
+import com.gildedrose.items.Ageable;
+import com.gildedrose.items.DefaultItem;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import waiter.Request;
+import waiter.Response;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ItemsControllerTest {
+
+    ArrayList<Ageable> mockedDB;
+    ItemsController itemsController;
+
+    @BeforeEach
+    void setUp() {
+        mockedDB = new ArrayList<>();
+        itemsController = new ItemsController(new ItemsDataSourceMock<>(mockedDB));
+        mockedDB.add(new DefaultItem(new Item("foo", 3, 14)));
+
```

Comment:
empty line

Reply:
Addressed in [this commit here](https://github.com/jramnani/GildedRoseJavaLLC/pull/24/commits/1983bd7ab5c6f63685323b51339047b4abb90ede)!